topics/groovy/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: groovy-lang
+created_by: James Strachan
+display_name: Groovy
+logo: groovy.png
+released: January 2, 2007
+short_description: Apache Groovy is an object-oriented programming language for the Java platform.
+topic: groovy
+url: http://groovy-lang.org/index.html
+wikipedia_url: https://en.wikipedia.org/wiki/Groovy_(programming_language)
+---
+Apache Groovy is a powerful, optionally typed and dynamic language, with static-typing and static compilation capabilities, for the Java platform aimed at improving developer productivity thanks to a concise, familiar and easy to learn syntax. It integrates smoothly with any Java program, and immediately delivers to your application powerful features, including scripting capabilities, Domain-Specific Language authoring, runtime and compile-time meta-programming and functional programming.
```

Comment:
The first sentence is pretty long, how about breaking it up? Maybe:

> Apache Groovy is a powerful, optionally typed and dynamic language for the Java platform. It features static typing and static compilation, as well as a concise, easy-to-learn syntax aimed at improving developer productivity.

Reply:
How about breaking up the second sentence and rephrasing a bit:

> It integrates smoothly with any Java program and provides scripting capabilities, Domain-Specific Language authoring, runtime and compile-time metaprogramming, and functional programming.

Reply:
Will update. Was using descriptions from the groovy website itself.